BioDrop/data/XanderRubio.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Alexander Clemens",
+    "bio": "Software Engineer | Open Source Maintainer | Colombia 🇨🇴",
+    "links": [
+        {
+            "name": "LinkedIn",
+            "url": "https://www.linkedin.com/in/alexanderclemens/",
+            "icon": "FaLinkedIn"
```

```suggestion
            "icon": "FaLinkedin"
```
FaLinkedIn icon isnot correct it must be either FaLinkedin or FaLinkedInin You can search for icons here https://www.biodrop.io/icons

---

Understood making the fix now.